site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -15373,28 +15403,36 @@ private void doHardDelete(String siteId) {
 	 * @return Map keyed on siteId. Set contains toolIds that have content.
 	 */
 	private Map<String, Set<String>> getSiteImportToolsWithContent(List<Site> sites, List<String> toolIds) {
-		
 		Map<String, Set<String>> siteToolsWithContent = new HashMap<>(); 
-		
+
 		for(Site site: sites) {
 			
 			Set<String> toolsWithContent = new HashSet<>();
 			
 			for(String toolId: toolIds) {
-				if(site.getToolForCommonId(toolId) != null) {
+				if(site.getToolForCommonId(toolId) != null ||
+						(("sakai.gradebook.tool".equals(toolId) || "sakai.gradebookng".equals(toolId)) &&
```

Comment:
Could simplify this by `StringUtils.equalsAny(toolId, "sakai.gradebook.tool", "sakai.gradebookng")`

Reply:
Ah nice.  Thanks Steve!  I'll make that change soon.